nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/TestValidateJson.java

```diff
@@ -109,6 +117,8 @@ void testPatternSchemaCheck() {
         runner.assertTransferCount(ValidateJson.REL_VALID, 0);
 
         assertValidationErrors(ValidateJson.REL_INVALID, true);
+        assertEquals(1, runner.getProvenanceEvents().size());
+        assertEquals(ProvenanceEventType.ROUTE, runner.getProvenanceEvents().get(0).getEventType());
```

Comment:
@MikeThomsen Thank you so much for reviewing and merging my code so quickly. I did have a question regarding the number of Provenance events with the invalid relationship. There is an attribute created when JSON is invalid which previously resulted in a Provenance modified attribute event. Now that I added the Provenance route event the modified attribute event is no longer there as this unit test indicates there is only one resulting Provenance event. Why is that so?